proposals/wac-ucr/index.bs

```diff
@@ -1111,16 +1532,259 @@ specially generated link to access the document with [=read access=]. He sets
+
+<div class="assertion">    
+  Related use cases: [[#basic-change-permissions]]
+</div>
+
+### The system shall provide the effective access permission modes on a certain [=resource=] or [=collection=] as they relate to the [=agent=] making the request, in the request response. ### {#req-effective-modes}
+
+<div class="assertion">    
+  Related use cases: [[#uc-client-determine-access-privileges]]
+</div>
+
+### The system shall limit the ability to read a certain [=resource=].### {#req-read}
+
+<div class="assertion">    
+  Related use cases: [[#basic-readonly]], [[#basic-write]], [[#basic-readappend]]
+</div>
+
+### The system shall limit the ability to change any of the existing contents of a certain [=resource=].### {#req-write}
+
+<div class="assertion">    
+  Related use cases: [[#basic-write]]
+</div>
+
+### The system shall limit the ability to change existing data in a certain [=resource=], such that only new data can be added to it.### {#req-append}
+
+<div class="assertion">    
+  Related use cases: [[#basic-appendonly]], [[#basic-readappend]]
+</div>
+
+### The system shall allow for access to a [=resource=] or [=collection=] to be limited to the [=agent=] that created it. ### {#req-creator}
 
-## Example Category ## {#example-category}
+<div class="assertion">    
+  Related use cases: [[#collection-readappendwrite]]
+</div>
 
-### This is an example requirement ### {#example-requirement}
+## Access to collections ## {#req-collections}
+    
+### The system shall limit the ability to read a certain [=collection=], exposing only the data from the [=collection=] resource itself, and a listing of its members. ### {#req-collection-read}
+
+<div class="assertion">    
+  Related use cases: [[#collection-readonly]], [[#collection-readwrite]], 
+  [[#collection-readappend]], [[#collection-readappendwrite]]
+</div>
+  
+### The system shall limit the ability to change data specific to a certain [=collection=], including only the data from the [=collection=] resource itself, and a listing of its members. ### {#req-collection-write}
+
+<div class="assertion">    
+  Related use cases: [[#collection-readwrite]]
+</div>  
+
+### The system shall limit the ability to create a [=resource=] in a certain [=collection=].### {#req-collection-create}
+
+<div class="assertion">    
+  Related use cases: [[#collection-readwrite]], [[#collection-readappend]],
+  [[#collection-readappendwrite]]
+</div>
 
-Assert: Related Use Cases: [[#uc-basic]]
+### The system shall allow for the creator of a [=resource=] or [=collection=] in a certain [=collection=] to be automatically granted access to the created [=resource=] or [=collection=].### {#req-collection-creator}
+
+<div class="assertion">    
+  Related use cases: [[#collection-readappendwrite]]
+</div>
+    
+### The system shall limit the ability to read the access permissions associated with a certain [=collection=].### {#req-collection-read-permissions}
+
```

Comment:
As explained above there are many situations where it makes sense to allow public read access to permissions. Eg: payment, age verification, drivers license requirements, etc.... 

"The system shall allow read access to permissions associated with a certain collection to be limited."

Reply:
Adjusted in https://github.com/solid/authorization-panel/pull/152/commits/691e677fb30249fd00b4d3a547899a98a7713f51